htsohm/htsohm.py

```diff
@@ -290,8 +484,26 @@ def calculate_percent_children_in_bin(run_id, generation, bin_coordinate):
         vf_bin=bin_coordinate[2]
     ).fetchall()
 
+    if (config['interactive_mode'] == 'on' and 
+            len([r for r in rows if r.in_bin]) != 0):
+        print('Number of children in the parent-bin :\t{}' \
+                .format(len([r for r in rows if r.in_bin])))
+        print('Total number of children :\t\t{}'.format(len(rows)))
+
     return len([ r for r in rows if r.in_bin ]) / len(rows)
 
+def are_you_sure(message, variable_name):
+    while True:
+        value = input(message)
+        proceed = input('Are you sure you want to set :\t{}\t=\t{}\t?'.format(variable_name, value))
+        if proceed in ['n', 'N']:
+            continue
+        elif proceed in ['y', 'Y']:
+            return value
+            break
+        else:
+            print('Please Enter y/n.')
+
```

Comment:
Try something like this:

def are_you_sure()
    while True:
        proceed = input('Are you sure? [y/n])
        if proceed in ['n', 'N']:
            return False
        elif proceed in ['y', 'Y']:
            return True        
        else:
            print('Please Enter y/n.')

def set_variable_better_name(message, variable_name):
    while True:
        value = input(message)
        print("Set var ### to ###")
        if are_you_sure():
            return value